inst/python_cmd.m

```diff
@@ -121,7 +121,7 @@
 %% Author: Colin B. Macdonald
 %% Keywords: python
 
-function varargout = python_cmd(cmd, varargin)
+function varout = python_cmd(cmd, varargin)
```

Comment:
I had to change this because if I do:
`python_cmd('return [10 10 10]')` Then I only get `ans=10`
same with `a = python_cmd('return [10 10 10]')` as `a=10`
.
Since now we are converting the returned objects into a cell array anyway, I thought it would be better to return the complete cell array and not just some element(s).
.
Is it okay. Or should I change it back to `varargout`?